storage/src/main/java/org/apache/uniffle/storage/handler/impl/MemoryClientReadHandler.java

```diff
@@ -47,15 +54,19 @@ public MemoryClientReadHandler(
       int shuffleId,
       int partitionId,
       int readBufferSize,
-      ShuffleServerClient shuffleServerClient) {
+      ShuffleServerClient shuffleServerClient,
+      Roaring64NavigableMap expectBlockIds,
+      Roaring64NavigableMap processBlockIds) {
```

Comment:
Could we lazily create memory handler?

Reply:
like https://github.com/apache/incubator-uniffle/blob/4b3d37db90fb54cc53a94480fa76982497b75844/storage/src/main/java/org/apache/uniffle/storage/handler/impl/ComposedClientReadHandler.java#L39

Reply:
We can create the handler when we need to use them and the memory handler don't need to care how to construct taskbitmap filter.

Reply:
> Could we lazily create memory handler?

Yes. Other handlers can be lazily created too. Should we raise a new pr?

Reply:
I think this pr need the lazy creation of memory handler. It will help us simplify the logic of this pr. You can implement it in another pr. And we merge that pr first . It's also ok to implement them in this pr at the same time.

Reply:
OK, i will do it in this pr